### PR TITLE
Fix button click

### PR DESCRIPTION
**Pull request by Fausto95**
Use a cb function instead

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/modal/index.js [hunk 1]
```diff
@@ -76,12 +76,12 @@ Modal.propTypes = {
     left: PropTypes.shape({
       title: PropTypes.string,
       subtitle: PropTypes.string,
-      link: PropTypes.string
+      onClick: PropTypes.string
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/molecule/modal/index.js]:
```suggestion
      onClick: PropTypes.func
```

File: packages/@coorpacademy-components/src/molecule/modal/index.js [hunk 2]
```diff
@@ -76,12 +76,12 @@ Modal.propTypes = {
...
     }),
     right: PropTypes.shape({
       title: PropTypes.string,
       subtitle: PropTypes.string,
-      link: PropTypes.string
+      onClick: PropTypes.string
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/molecule/modal/index.js]:
```suggestion
      onClick: PropTypes.func
```

File: packages/@coorpacademy-components/src/molecule/modal/index.js [hunk 3]
```diff
@@ -13,9 +13,9 @@ import PropTypes from 'prop-types';
 import Provider from '../../atom/provider';
 import style from './style.css';
 
-const Choice = ({title, subtitle, link, children}) => {
+const Choice = ({title, subtitle, onClick, children}) => {
```
Comment (VincentCATILLON) [inline packages/@coorpacademy-components/src/molecule/modal/index.js]: why do we have two components in the same file ?

**Comment by Fausto95** [comment]
> Because "history"

I feel you, sorry :(

**Pull request merged**